### PR TITLE
Event tests

### PR DESCRIPTION
**Pull request by tdfischer**

### CODE REVIEW

File: tests/events.rs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        let mut sbox = sandbox::Sandbox::new(Box::new(exec), &mut watcher);
+        sbox.spawn();
+        sbox.tick();
+    }
+
+    assert!(watcher.hit);
+}
+
+// For some reason, this can hang.
+//#[test]
```
Comment (wilsonianb) [inline tests/events.rs:42]: Is this test intended to be skipped for now?

Comment (tdfischer) [inline tests/events.rs:42]: Yes. It hangs for weird reasons that aren't related to the event pumping stuff.

**Comment by wilsonianb** [comment]
LGTM

**Comment by codius-montecarlo** [comment]
The build is good and I'm seeing 1 +1s from the following folks:
- wilsonianb

As such, I'll attempt to merge and close this pull request.

**Pull request merged**